vagrant_provision.sh

```diff
@@ -52,9 +52,21 @@ cat > /home/vagrant/hadoop3/etc/hadoop/hdfs-site.xml <<EOF
     </property>
 </configuration>
 EOF
+
+# add custom config files to Hadoop
+cat /home/vagrant/rdfs/config/hdfs-site.xml > /home/vagrant/hadoop/etc/hadoop/hdfs-site.xml
+cat /home/vagrant/rdfs/config/core-site.xml > /home/vagrant/hadoop/etc/hadoop/core-site.xml
```

Comment:
I think they're touching different files (the ones in /home/vagrant/hadoop3 as opposed to /home/vagrant/hadoop). It looks like they might be unnecessary anyway, but I'm not totally sure